pkg/os/shell/shell_windows.go

```diff
@@ -30,6 +30,7 @@ func getProcessEntry(pid int) (pe *syscall.ProcessEntry32, err error) {
 	}
 
 	for {
+		//#nosec G115 -- pid is coming from os.Getppid() as int
```

Comment:
the `os.Getppid()` call is 2 level above `getProcessEntry` in the call chain. Nothing guarantees that in the future we are not going to reuse this function with a `pid` value coming from something different.
I'd prefer to change the `getProcessEntry(pid int)` to `getProcessEntry(pid uint32)`, and to change `detect` to convert the value returned by `os.Getppid()` to `uint32`